fonts.css

```diff
@@ -0,0 +1,61 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap");
+
+@font-face {
+  font-family: "Montserrat";
+  src:  url("../fonts/Montserrat.woff") format("woff"),
+  url("../fonts/montserrat-v14-latin_cyrillic-regular.woff2") format("woff2");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/itog/pullRequest?prid=5556249)